tests/Oryx.BuildImage.Tests/CommandTests/OryxCommandTest.cs

```diff
@@ -140,10 +140,10 @@ public void BuildImage_CanExec_SingleCommand()
                 () =>
                 {
                     Assert.Contains("#!" + expectedBashPath, result.StdOut);
-                    Assert.Contains($"{NodeConstants.PlatformName}={NodeConstants.NodeLtsVersion}", result.StdOut);
+                    Assert.Contains($"{NodeConstants.PlatformName}={NodeVersions.Node14Version}", result.StdOut);
                     Assert.True(result.IsSuccess);
                     // Actual output from `node --version` starts with a 'v'
-                    Assert.Contains($"v{NodeConstants.NodeLtsVersion}", result.StdOut);
+                    Assert.Contains($"v{NodeVersions.Node14Version}", result.StdOut);
```

Comment:
@qianz2 should we keep `NodeConstants.NodeLtsVersion` so that it tests against the new LTS (node16)?

Reply:
@william-msft It looks that for this test, the result will still be node 14, I'm trying to get a deeper look why it happens.